server/rest/src/main/java/org/infinispan/rest/resources/TasksResource.java

```diff
@@ -64,10 +64,15 @@ public Invocations getInvocations() {
    }
 
    private CompletionStage<RestResponse> createScriptTask(RestRequest request) {
+      NettyRestResponse.Builder builder = new NettyRestResponse.Builder();
```

Comment:
appart from REST, there is no other place to create a task where the name validation should be added?

Reply:
I don't know for sure, it appears the only way is to get hold of a ScriptManager and register a task

Reply:
This will only be possible for embedded though. Hot Rod does not allow the creation of tasks

Reply:
From the REST POV, we're good. We cannot delete tasks so far, only create, list and execute

Reply:
ok, done